lib/middleware/lambdas.js

```diff
@@ -20,4 +20,5 @@ module.exports = function lambdas(req, res) {
       };
     }
   });
+  next();
 };
```

Comment:
https://github.com/UKHomeOfficeForms/hof-form-wizard/blob/master/lib/middleware/session.js

we include options if we need to, I don't see the point of defining options and doing nothing with it - that would actually fail linting